unit_tests/test_tester.py

```diff
@@ -55,6 +53,17 @@ def _init_params(self):
     @pytest.fixture(autouse=True, name='setup_logging')
     def fixture_setup_logging(self, tmp_path):
         self._init_logging(tmp_path / self.__class__.__name__)
+        self.kafka_cluster = None
+
+    @pytest.fixture(scope="function", autouse=True)
+    def fixture_mock_issues(self):
+        """
+        In Pytester the boto credentials are not set, so we mock the issue details, so we dont need them.
```

Comment:
if need we can pass in the credentials (i.e. in CI it would pass it in)

Reply:
I do not think it is needed. I did not delve deeper why the credentiasl are not passed, but given what they are for, I think skiping more calls to AWS is better for a unit test